app/src/main/java/org/schabi/newpipe/report/ErrorInfo.kt

```diff
@@ -5,7 +5,7 @@ import androidx.annotation.StringRes
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
-class ErrorInfo private constructor(
+class ErrorInfo constructor(
```

Comment:
The `constructor` keyword can be removed as well.

Reply:
good catch